### PR TITLE
updating new jmeter versions and setting new default

### PR DESCRIPTION
**Pull request by xshyamx**
Update with new JMeter versions and set the default to the latest version

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/JMeterMigration.scala [hunk 1]
```diff
@@ -2,21 +2,41 @@ package io.sdkman.changelogs
 
 import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
 import com.mongodb.client.MongoDatabase
+import org.bson.Document
 
 @ChangeLog(order = "051")
 class JMeterMigration {
+  val CandidateName  = "jmeter"
+  val DefaultVersion = "5.4.3"
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JMeterMigration.scala:10]: You can probably ditch these constants, as they will only ever be used in one place and make things more confusing to grok.

Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/JMeterMigration.scala]: Removed

File: src/main/scala/io/sdkman/changelogs/JMeterMigration.scala [hunk 2]
```diff
@@ -2,21 +2,39 @@ package io.sdkman.changelogs
...
+            version = version,
+            url =
+              s"https://archive.apache.org/dist/jmeter/binaries/apache-jmeter-$version.zip"
+          )
+      )
+      .validate()
+      .insert()
+  }
   @ChangeSet(
-    order = "001",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JMeterMigration.scala:9]: Sorry to be a pain, but let's keep the main candidate migration here for reference like we've done everywhere else. We're only removing version migrations for brevity.

Comment (xshyamx) [reply src/main/scala/io/sdkman/changelogs/JMeterMigration.scala:9]: No problem. I understand the reasoning behind :). Updated the PR

**Review by marc0der — APPROVED** [review]
Thanks for contributing!

**Pull request merged**